readme.md

```diff
@@ -4,15 +4,15 @@
 ![Banner](wp-assets/banner-1544x500.png)
 Enable AMP on your WordPress site, the WordPress way.
 
-**Contributors:** [automattic](https://profiles.wordpress.org/automattic), [xwp](https://profiles.wordpress.org/xwp), [google](https://profiles.wordpress.org/google), [westonruter](https://profiles.wordpress.org/westonruter), [ryankienstra](https://profiles.wordpress.org/ryankienstra), [batmoo](https://profiles.wordpress.org/batmoo), [stubgo](https://profiles.wordpress.org/stubgo), [albertomedina](https://profiles.wordpress.org/albertomedina), [tweetythierry](https://profiles.wordpress.org/tweetythierry), [joshuawold](https://profiles.wordpress.org/joshuawold), [postphotos](https://profiles.wordpress.org/postphotos)  
-**Tags:** [amp](https://wordpress.org/plugins/tags/amp), [mobile](https://wordpress.org/plugins/tags/mobile)  
-**Requires at least:** 4.9  
-**Tested up to:** 5.1  
-**Stable tag:** 1.0.2  
-**License:** [GPLv2 or later](http://www.gnu.org/licenses/gpl-2.0.html)  
-**Requires PHP:** 5.4  
```

Comment:
You should probably fix your IDE configuration to not strip whitespace at the end of the line. This is required here for Markdown and thus part of the `.editorconfig`: https://github.com/ampproject/amp-wp/blob/cd31949d1b49c0e2eae42c2c1efc15a0c1cf9862/.editorconfig#L17-L18